ansible/cluster.yml

```diff
@@ -65,6 +71,7 @@
 # install kubernetes on the nodes
 - hosts: nodes
   sudo: yes
+  serial: 5
```

Comment:
Good point. This is only required when the cluster has many nodes (I tested with 20). If it will be helpful, I can take it out of this PR.